resource-definitions/terraform-driver/runner-pod-configuration/k8s-cluster-refs.tf

```
@@ -0,0 +1,30 @@
+resource "humanitec_resource_definition" "aks_resource_cluster" {
+  id          = "eks-enabled-cluster"
```

```suggestion
  id          = "aad-enabled-cluster"
```
Do we mean "AAD enabled"? Might be a copy&paste effect.